pages/login.page.js

```diff
@@ -0,0 +1,26 @@
+import BasePage from '../pages/base.page';
+
+class LoginPage extends BasePage{
+
+    get userNameInput(){return $('#username')}
+    get passWordInput(){return $('#password')}
+    get loginButton(){return $('#submit')}
+    get rememberMe(){return $('#remember-me')}
+    get statusMessage(){return $('#badge badge-pill badge-danger')}
```

Comment:
You don't need the whole chain of badges to select this. Prefer only `.badge` (noticed I use a class selector instead of an id based, as you used, because badge is a class in this case, so `#badge` does not exist in the page), as the rest of them capture style specific stuff that you don't need for individualizing the element.